capa/features/extractors/cape/call.py

```diff
@@ -50,7 +51,8 @@ def extract_call_features(ph: ProcessHandle, th: ThreadHandle, ch: CallHandle) -
         else:
             assert_never(value)
 
-    yield API(call.api), ch.address
+    for name in capa.features.extractors.helpers.generate_symbols("", call.api):
```

Comment:
we should update this API to accept None when the dll isn't present, not the empty string. But that can happen in another PR.